static/css/s2.css

```diff
@@ -13161,6 +13161,11 @@ span.ref-link-color-3 {color: blue}
   padding: 3px 10px;
 }
 
+.stickySidebarFooter {
```

Comment:
Where are we sticking this to with top:0?

Reply:
It's how I managed to get the stickiness working... based on an article I can no longer find... unfortunately. Would you suggest another approach?

Reply:
Oh, I'm sorry - I misread the `where` as a `why`. My bad! 

We are sticking it to the top of the sidebar nav when you scroll down, as per the specs.